src/com/epam/javatrainings/classdecomposition/pizzeria_palmetto/services/OrderCreator.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.javatrainings.classdecomposition.pizzeria_palmetto.services;
+
+import com.epam.javatrainings.classdecomposition.pizzeria_palmetto.model.Pizza;
+
+public interface OrderCreator {
+    Pizza create(String name, String type, int quantity);
```

Comment:
"String type" should be replaced with "Type type"

Reply:
done